cloth-diapers/wash-routine.md

```diff
@@ -0,0 +1,139 @@
+# Wash routine
+
+> The wash routine for cloth diapers is the **main source** of _every_ cloth diaper issue from smells to rashes to leaks.
+
+The cloth diaper groups online are ***filled*** with people bemoaning issues with their cloth diapers that turn into nothing but 'let's trouble shoot your wash routine' posts.
+
+***The problem is simple: there is a HUGE amount of misinformation out
```

Comment:
The formatting of this intro could be re-worked/worded ...
![image](https://user-images.githubusercontent.com/194400/173181110-28f78bff-5b50-442a-bc31-87baeff411a6.png)